### PR TITLE
aa-complain: add German translation

### PR DESCRIPTION
**Pull request by TimGoldbach**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thanks for your contribution.

File: pages.de/linux/aa-complain.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# aa-complain
+
+> Setze eine AppArmor-Richtlinie in den Beschwerde-Modus.
+> Siehe auch: `aa-disable`, `aa-enforce`, `aa-status`.
+> Weitere Informationen: <https://gitlab.com/apparmor/apparmor/-/wikis/manpage_aa-complain.8>.
+
+- Setze eine Richtlinie in den Beschwerde-Modus:
+
+`sudo aa-complain {{pfad/zum/profil}}`
```
Comment (blueskyson) [inline pages.de/linux/aa-complain.md:9]: According to the translation [template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md).
```suggestion
`sudo aa-complain {{pfad/zu/profil}}`
```

**Comment by kbdharun** [comment]
@TimGoldbach Can you sign the CLA for this PR to go through? (Thanks in advance)

File: pages.de/linux/aa-complain.md (hunk 1 above)
Comment (pixelcmtd) [reply pages.de/linux/aa-complain.md:9]: Not sure whether the translation template applies here. I think [we have somewhat consistently used „zum“ where grammatically correct](https://github.com/search?q=repo%3Atldr-pages%2Ftldr+path%3A%2F%5Epages%5C.de%5C%2F%2F+pfad%2Fzum&type=code), but some of those are also `pfad/zum/verzeichnis` which the template explicitly forbids.

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

**Comment by kbdharun** [comment]
@TimGoldbach Any updates on this?

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by gutjuri** [comment]
> Shall we take over this PR to merge this?

How can we? If the original author doesn't sign the CLA we'd have to rewrite the page from scratch in order not to infringe on copyright if I'm not mistaken.

**Pull request closed without merging**